### ISSUE TITLE
Control is not meeting the Accessibility rules

### ISSUE THREAD

**Issue by Pradeep-K-P** [srigar/multiselect-react-dropdown#192]
We are using this control in one of the application that we support. As part of Accessibility check, it is mentioned that this control is not meeting the accessibility rules.

Title : Screen reader is not announcing any information while navigating, selecting & removing the controls inside the edit field dropdown.
User Experience: Users who rely on Assistive technology will face difficulty and unable to access information as screen reader is not announcing any information while navigating, selecting & removing the suggestions for edit field due to this users cannot get information about the control. 
Pre-requisite: Turn on NVDA (Download link : https://www.nvaccess.org/download/)
Repro Steps: 
	Step 1: Open the web page where this control is implemented in the Chrome browser. 
	Step 2: Navigate to the page with tab key. 
	Step 3: Verify whether Screen reader is not announcing any information while navigating, selecting & removing the controls inside the edit field dropdown or not. 
	
Actual Result: Screen reader is not announcing any information while navigating, selecting & removing the controls inside the edit field dropdown. 
Current Observation: While navigating through search suggestions screen reader is announcing as blank & when we select/ remove  the suggestions screen reader is not announcing any information, screen reader is silent.  
Expected Result: Screen reader should announce information while navigating, selecting & removing the controls inside the edit field dropdown. 
Expected behavior: While navigating through search suggestions screen reader should announce information as  "expanded list with 6 items 1 of 6 all check box unchecked" & when we select/ remove the suggestions screen reader should announce as all selected & when we remove screen reader should announce as all removed 
Observation: Same issue is observed in New Edge Browser. 
Narrator Behavior: Screen reader silent while navigating, selecting & removing the suggestions for edit field.

**Comment by mishra1010** [comment]
@srigar Please let us know if we can have any changes to make this accessibility controls available

**Comment by srigar** [comment]
Will check and update you timeline

**Comment by mishra1010** [comment]
Thanks

**Comment by supriya-kajana** [comment]
Hi @srigar any update on fixing accessibility issues for the control
